rudra/deployments/emrs/pypi_emr.py

```diff
@@ -0,0 +1,52 @@
+"""EMR script implementation for the PYPI service."""
```

Comment:
All these EMR files looks the same. Can this not be clubbed further?

Reply:
Yes, for now,  EMR files looks the same but we have to customize these files according to the model requirements.